src/EnergyPlus/HeatingCoils.cc

```diff
@@ -163,7 +163,7 @@ namespace HeatingCoils {
             if (CompIndex == 0) {
                 CoilNum = UtilityRoutines::FindItemInList(CompName, state.dataHeatingCoils->HeatingCoil);
                 if (CoilNum == 0) {
-                    ShowFatalError(state, "SimulateHeatingCoilComponents: Coil not found=" + std::string{CompName});
+                    ShowFatalError(state, format("SimulateHeatingCoilComponents: Coil not found={}", std::string{CompName}));
```

Comment:
`format` statements are nice to see in place of string concatenation.  The actual impact of it in this case is maybe not too much since it is only a single concatenation, but still it's good practice.  However, you can/should remove the `std::string` constructor, and just pass `CompName` directly as an arg to the format function. Otherwise you are creating a string object for no reason, which is bad practice.